tests/integration/test_fim/test_files/test_report_changes/test_report_changes_and_diff.py

```diff
@@ -66,67 +65,54 @@
     - fim_report_changes
 '''
 import os
-import re
-import sys
+
 
 import pytest
+from test_fim.test_files.test_report_changes.common import make_diff_file_path
```

Comment:
If you see that the method is used outside of the report_changes test you should move it, or copy it to a place where will use it for the fim module and start using that. If not, the framework continues to get dirty.

Reply:
fixed in https://github.com/wazuh/wazuh-qa/pull/3503/commits/7c5a46a36366ade14be090824ef103620354c2c5